apps/web/src/app/(pages)/(dashboard)/roles/page.tsx

```diff
@@ -22,6 +22,9 @@ export default function Roles({
   searchParams,
 }: {
   searchParams?: {
+    workTerm?: WorkTermType;
+    workEnvironment?: WorkEnvironmentType;
+    search?: string;
     cycle?: WorkTermType;
```

Comment:
it looks like we want to get rid of `cycle` and `term` now since you renamed them in the new search params.

Reply:
Yeah I think we got rid of `workTerm` and `workEnvironment` in favor of `cycle` and `term` but I might be wrong. This might an artifact of a merge. @suxls would recommend removing `workTerm` and `workEnvironment` from the `searchParams` object and testing.